winterfell/services/claimService.js

```diff
@@ -5,19 +5,11 @@ var Claim = require('./../models/claim');
 
 function ClaimService(app) {
   this.app = app;
-};
+}
 
 module.exports = ClaimService;
 module.exports.createNewClaim = function(userId, callback) {
-  return Claim.findOne({ userId: userId, state: Claim.State.INCOMPLETE }).exec(function(err, fileClaim) {
```

Comment:
I guess it's a bit weird that now this is reduced to one simple statement.
I don't think we should ever have more than one INCOMPLETE claim. `Claim.quickCreate()` is the part you described that just creates a new claim. However, if multiple controllers / services want to create a claim, they should have the logic to check existing incomplete claims. Rather than having this logic in all those other places, I think it should be consolidated here.